airbyte-workers/build.gradle

```diff
@@ -77,3 +77,22 @@ task copyGeneratedTar(type: Copy) {
 Task dockerBuildTask = getDockerBuildTask("worker",  "$project.projectDir")
 dockerBuildTask.dependsOn(copyGeneratedTar)
 assemble.dependsOn(dockerBuildTask)
+
+
+test {
+    useJUnitPlatform {
+        excludeTags 'log4j2-config', 'cloud-storage'
```

Comment:
The excluded tags can go into the root gradle. There are already some tags being excluded.

Reply:
should everything in `airbyte-config/models/build.gradle` be moved as well?

Reply:
Yes so it is defined in a single place for the exclusion.

Reply:
done.